lib/python/picongpu/plugins/plot_mpl/requirements.txt

```diff
@@ -1,2 +1,3 @@
 matplotlib
 numpy
+-r jupyter_widgets/requirements.txt
```

Comment:
it's the other way around. `jupyter_widgets/requirements.txt` depends on `plot_mpl/requirements.txt`